src/releaseBusV2/release-bus-v2.github-app.ts

```
@@ -514,9 +519,19 @@ export class ReleaseBusGitHubApp {
     pull: GitHubPullRequestDetails
   ): Promise<readonly string[]> {
     const logins: string[] = [];
-    const addLogin = (value: string | undefined) => {
-      const login = value?.trim();
-      if (!login || isReleaseBusGitHubAppActor(login)) return;
+    const addUser = (
+      value:
+        { readonly login?: string; readonly type?: string } | null | undefined
```

ESLint

[error] Replace `{·readonly·login?:·string;·readonly·type?:·string·}·|·null` with `|·{·readonly·login?:·string;·readonly·type?:·string·}⏎········|·null⏎·······`

(prettier/prettier)